BSPConversionLib/Source/EntityConverter.cs

```diff
@@ -187,14 +195,20 @@ private static void OpenDoorOnPressed(Entity button, Entity door)
 
 		private static void SetButtonFlags(Entity button)
 		{
+			var spawnflags = 0;
+
 			if (!float.TryParse(button["speed"], out var speed))
 				return;
 
 			if ((speed == -1 || speed >= 9999) && (button["wait"] == "-1")) // TODO: Add customization setting for the upper bounds potentially?
-				button["spawnflags"] = "1"; // Don't move flag
+				spawnflags += (int)FuncButtonFlags.DontMove;
```

Comment:
Use `|=` when assigning flags.
`spawnflags |= (int)FuncButtonFlags.DontMove;`

Reply:
Sure, what does this do exactly?